recipes/python-quickjs/bld.bat

```diff
@@ -0,0 +1,3 @@
+"%PYTHON%" setup.py build -c mingw32
```

Comment:
Do we still need this? If not we could get rid of the build scripts.

Reply:
This one is needed otherwise we fallback to MVSC compiler.